index.html

```diff
@@ -687,15 +907,25 @@ <h3>
           algorithm's structure to reflect the intended parallel execution,
           improving clarity and aligning with implementation expectations.
         </aside>
+        <aside class="addition" id="a11">
+          <span class="marker">Candidate Addition:</span> Update request
+          position algorithm for approximate location support.
+        </aside>
         <ol class="algorithm">
           <li>Let |watchIDs:List| be |geolocation|'s
           {{Geolocation/[[watchIDs]]}}.
           </li>
           <li>Let |document:Document| be the |geolocation|'s [=relevant global
           object=]'s [=associated `Document`=].
           </li>
+          <li>
+            <ins cite="#a11">Let |permissionName| be "geolocation-approximate"
```

Comment:
"geolocation-approximate" and "geolocation" should link to the definitions in "Checking permission to use the API"